examples/resources/snowflake_role_grants/import.sh

```diff
@@ -1,2 +0,0 @@
-# format is role_name|roles|users
-terraform import snowflake_role_grants.example "role_name|role1,role2|user1,user2"
```

Comment:
It's a lot of files, but to confirm was examples/resources/snowflake_role_ownership_grant removed ?

Reply:
yes, missed the example, though, now removed too